src/style.css

```diff
@@ -1,3 +1,64 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family for Roboto to ensure text is displayed with a default sans-serif font if Roboto fails to load. For example, 'font-family: Roboto, Arial, sans-serif;'.